masschallenge/django-accelerator
# [AC-5459] Data migration to add mc related permissions

To test:
- Get code on development and reload the database:
```
cd accelerate
make checkout
make load-db
make run-server
```
- You should now be able to reproduce the problem with:
-- Masquerade as Ryan: http://localhost:8181/admin/simpleuser/user/47981/masquerade/
-- Goto http://localhost:8181/admin/mc/newsletter/967/
-- At the bottom of the page there should be no cut off "Change" link under "Sections" (see ticket for what it should look like)
-- Goto http://localhost:8181/admin/mc/newsletter/10077/
-- There should be no startup team members at the bottom of the page.

- To see the fix:
```
make stop-server  # Ctrl-C in the server window
cd ../django-accelerator
git checkout AC-5459
cd ../accelerate
make migrate
make run-server
```
- You should no longer see the problem if you go through the above process.
